src/game.cc

```diff
@@ -94,35 +97,35 @@ void Game::PrintInfoFrame()
 }
 
 void Game::PrintPrepScreen()
-{  
+{
   char* mode;
   const char* kStartArt = R"ronmak(
```

Comment:
![image](https://user-images.githubusercontent.com/36345325/95025214-f8fbfb80-063c-11eb-895f-2ad18ded2754.png)

Reply:
The man himself. I have both him and this project to thank as part of getting me into grad school